<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>1081</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #999;
border-radius: 5px;
padding: 8px 15px;
font-weight: bold;</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>511</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Играть</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Правила</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Режимы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>611</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #d4d4d4;
color: white;
border-radius: 5px;
padding: 5px;
font-weight: bold;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>511</width>
          <height>641</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Цель игры:
Создать плитку с числом  &quot;2048&quot;, объединяя одинаковые числа. 
Как играть:
Игровое поле – это сетка 4×4 или 6×6 .В начале игры появляются две плитка со значением &quot;2&quot; или &quot;4&quot; в случайных местах и в процессе игры после каждого хода.Используй стрелки (↑, ↓, ←, →) или клавиши клавиатуры (W,A,S,D) чтобы сдвигать все плитки в выбранном направлении.Плитки двигаются до упора, пока не упрутся в край поля или другую плитку.  
Как складываются плитки:
Если две одинаковые плитки сталкиваются, ониобъединяются в одну.Например: 2 + 2 = 4, 4 + 4 = 8, и так далее. За один ход плитка может объединиться только один раз.Например, если в ряду стоят &quot;2&quot;, &quot;2&quot;, &quot;2&quot;, то после сдвига получится &quot;4&quot;,&quot;2&quot;
Как считается счёт:
Каждый раз, когда две плитки объединяются, их сумма добавляется к твоему счёту. 
Когда ты выиграл или проиграл:
Если на поле появилась плитка 2048&quot;,ты победил!Можно продолжать играть,чтобы набрать очки,создать новые плитки.Если поле полностью заполнено и нет возможных ходов, игра заканчивается.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>509</width>
          <height>78</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Правила игры</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>471</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #999;
border-radius: 5px;
padding: 8px 15px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>450</y>
          <width>461</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #999;
border-radius: 5px;
padding: 8px 15px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>380</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>4 X 4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>670</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>6 X 6</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>180</y>
          <width>261</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Классический вариант игры 2048. Играй в свое удововльствие!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>470</y>
          <width>261</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Больше поле — больше возможностей</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>431</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Режимы игры</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>471</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Настройки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Громкость эффектов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="styleSheet">
            <string notr="true">background-color: #999;
border-radius: 5px;
padding: 8px 15px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Сброс данных</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
